.circleci/config.yml

```diff
@@ -317,9 +317,12 @@ jobs:
       - run:
           name: Run e2e tests (Puppeteer and Jest)
           working_directory: ~/workbench/e2e
-          command: yarn --silent test:ci
+          command: yarn test:ci
       - store_artifacts:
           path: e2e/logs
+          destination: logs
+      - store_test_results:
+          path: ~/workbench/e2e/logs
```

Comment:
Just to check, this is writing to circle, right? and nowhere else?

Reply:
In `e2e` project, any screenshot or/and html file are saved in `/logs` sub-directory. If you run tests on localhost, you'll find failed screenshots and html files in `/logs` dir. 
We have to set `store_test_results` key to tell CircleCI where to find saved files.